.github/CODEOWNERS

```
@@ -1,11 +1,15 @@
 # Default owners for everything
-* @suzubara @abbyoung @esacteksab
+
+- @suzubara @abbyoung @esacteksab @gidjin
```

I think this should be:

```suggestion
* @suzubara @abbyoung @esacteksab @gidjin
```